src/Yoti.Auth/DigitalIdentity/ShareSessionResult.cs

```diff
@@ -1,14 +1,11 @@
-﻿using System;
-using System.Reflection.Emit;
-using Newtonsoft.Json;
-using Yoti.Auth.DataObjects;
+﻿using Newtonsoft.Json;
+
 
 namespace Yoti.Auth.DigitalIdentity
 {
     public class ShareSessionResult
```

Comment:
`status` is missing

Reply:
I have checked and status is in ShareSessionResult file.  So it seems there is no problem